### PR TITLE
[#11878] Add new account request alert email for admins

### PR DESCRIPTION
**Pull request by jayasting98**
Part of #11878

**Outline of Solution**

 

Added the new account request alert email for admins. If `comments` is null, we leave the table entry blank.

The following is a preview of the email template.

image

### CODE REVIEW

**Review by EuniceSim142 — APPROVED** [review]
lgtm, thanks!

File: src/main/java/teammates/common/util/EmailType.java [hunk 1]
```diff
@@ -23,6 +23,7 @@ public enum EmailType {
     NEW_INSTRUCTOR_ACCOUNT("TEAMMATES: Welcome to TEAMMATES! %s"),
     STUDENT_COURSE_JOIN("TEAMMATES: Invitation to join course [%s][Course ID: %s]"),
     STUDENT_COURSE_REJOIN_AFTER_GOOGLE_ID_RESET("TEAMMATES: Your account has been reset for course [%s][Course ID: %s]"),
+    NEW_ACCOUNT_REQUEST_ADMIN_ALERT("TEAMMATES: New Account Request Received"),
```
Comment (ziqing26) [inline src/main/java/teammates/common/util/EmailType.java:26]:
```suggestion
    NEW_ACCOUNT_REQUEST_ADMIN_ALERT("TEAMMATES (Action Needed): New Account Request Received"),
```

Comment (ziqing26) [reply src/main/java/teammates/common/util/EmailType.java:26]: Will this be more telling for admin to take actions

**Review by ziqing26 — APPROVED** [review]
LGTM! Only one point of discussion 🙏

File: src/main/java/teammates/common/util/EmailType.java (hunk 1 above)
Comment (jayasting98) [reply src/main/java/teammates/common/util/EmailType.java:26]: I am supportive of this change. However, do we still need to check with the prof about this? If not, I can just make this change.

Comment (ziqing26) [reply src/main/java/teammates/common/util/EmailType.java:26]: Yep let's change it. Thanks!

**Review by ziqing26 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**